tests/conftest.py

```diff
@@ -97,23 +96,13 @@ def pytest_addoption(parser, pluginmanager) -> None:  # pylint: disable=unused-a
 def fixture_validate_options(request) -> None:
     """This fixture only exists to validate the custom command line flags."""
     kafka_bootstrap_servers = request.config.getoption("kafka_bootstrap_servers")
-    log_dir = request.config.getoption("log_dir")
-    kafka_version = request.config.getoption("kafka_version")
     registry_url = request.config.getoption("registry_url")
     rest_url = request.config.getoption("rest_url")
     server_ca = request.config.getoption("server_ca")
 
     has_external_registry_or_rest = registry_url or rest_url
 
-    if not re.match(VERSION_REGEX, kafka_version):
```

Comment:
Now this is causing a lint error:

```
tests/conftest.py:6:1: F401 're' imported but unused
```

Please run `pre-commit` there are also some issues with the `.sh` file